network/hostnetwork/packet/request.go

```diff
@@ -32,6 +32,11 @@ const (
 	RevokeAuth
 )
 
+// RequestCheckNodePriv is data for check node privileges.
+type RequestCheckNodePriv struct {
+	PrivKey string
```

Comment:
comment what is  PrivKey, somebody may think it's private key instead privilegie key